src/Components/Aside/styles.css

```diff
@@ -0,0 +1,48 @@
+.aside-container {
+  background-color: var(--background-color);
+  color: var(--secondary-color);
+  width: 15rem;
+  height: 44rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+
+  border: 2px solid black;
+  border-right: 3px solid var(--secondary-color);
+}
```

Comment:
Dimensions for sidebar can be set by using grid and percentages to make it more responsive.

Reply:
I've added responsiveness